### PR TITLE
Specify special configs to run tests faster

### PR DESCRIPTION
**Pull request by zurk**
Closes https://github.com/src-d/style-analyzer/issues/432

### CODE REVIEW

File: lookout/style/format/tests/test_analyzer.py [hunk 1]
```diff
@@ -19,6 +20,35 @@
 Change = NamedTuple("Change", [("base", File), ("head", File)])
 
 
+json_train_config = {
```
Comment (m09) [inline lookout/style/format/tests/test_analyzer.py]: If we wrap this dict into a function we'll avoid any chance of editing it in some place and creating weird bugs (even though right now everything is deepcopy-ed).

Comment (vmarkovtsev) [reply lookout/style/format/tests/test_analyzer.py]: +1

File: lookout/style/format/tests/test_analyzer.py [hunk 2]
```diff
@@ -118,20 +148,21 @@ def test_analyze(self):
...
-        config = {"global": {"n_iter": 1, "cutoff_label_precision": 0.8,
-                             "feature_extractor": {"cutoff_label_support": 50}}}
-        model = FormatAnalyzer.train(self.ptr, config, self.data_service)
-        analyzer = FormatAnalyzer(model, self.ptr.url, {"confidence_threshold": 0.90,
-                                                        "support_threshold": 50})
+        config = get_train_config()
+        # Make uast_break_check only here
+        config["uast_break_check"] = True
+        model = FormatAnalyzer.train(self.ptr, get_train_config(), self.data_service)
+        analyzer = FormatAnalyzer(model, self.ptr.url, get_analyze_config)
```
Comment (m09) [inline lookout/style/format/tests/test_analyzer.py]: lacks ()

File: lookout/style/format/tests/test_quality_report.py [hunk 3]
```diff
@@ -106,10 +107,11 @@ def test_train_review_analyzer_integration(self):
         with TestAnalyzer(port=self.port, db=self.db.name, fs=self.fs.name,
                           analyzer="lookout.style.format.quality_report"):
             server.run("push", FROM_COMMIT, TO_COMMIT, port=self.port,
-                       git_dir=self.jquery_dir)
+                       git_dir=self.jquery_dir, config_json=json.dumps({
+                            QualityReportAnalyzer.name: get_train_config()}))
             server.run("review", FROM_COMMIT, TO_COMMIT, port=self.port,
                        git_dir=self.jquery_dir, config_json=json.dumps({
-                            QualityReportAnalyzer.name: {"uast_break_check": False}}))
+                            QualityReportAnalyzer.name: get_analyze_config}))
```
Comment (m09) [inline lookout/style/format/tests/test_quality_report.py]: lacks ()

Comment (zurk) [reply lookout/style/format/tests/test_quality_report.py]: sorry 😅

**Pull request merged**